source/fundamentals/data-formats/document-data-format-record.txt

```diff
@@ -100,36 +105,10 @@ as shown in the following code:
 Specify Component Conversion Using Annotations
 ----------------------------------------------
 
-This section describes the annotations you can use to configure the
-serialization behavior of record components and provides an example to
-demonstrate the annotation behavior.
-
-.. tip::
-
-   The ``org.bson.codecs.records.annotations`` package is deprecated. Use the
-   equivalent ones from the `org.bson.codecs.pojo.annotations <{+api+}/apidocs/bson/org/bson/codecs/pojo/annotations/package-summary.html>`__
-   package instead.
-
-You can use the following annotations on record components:
-
-.. list-table::
-   :header-rows: 1
-   :stub-columns: 1
-   :widths: 10 10
-
-   * - Annotation Name
-     - Description
-
-   * - ``BsonId``
-     - Specifies the component to serialize as the _id property.
-
-   * - ``BsonProperty``
-     - Specifies a custom document field name when converting the record
-       component to BSON. Accepts the field name as the parameter.
-
-   * - ``BsonRepresentation``
-     - Specifies a BSON type to store when different from the record component
-       type. Accepts the BSON type as the parameter.
```

Comment:
Q: do you need to delete this table? It looks like these annotations (BsonId, BsonProperty, and BsonRepresentation) are still supported in the `org.bson.codecs.pojo.annotations` package

Reply:
They are still supported, but it makes more sense to rely on the API documentation that will be updated and subject to change without us necessarily knowing about it. I know they're used in the examples, but I think encouraging users to view the whole list of annotations is beneficial.

Reply:
Makes sense, thanks for clarifying!